.github/workflows/.vib/vib-pipeline.json

```diff
@@ -31,22 +31,17 @@
           }
         },
         {
-          "action_id": "grype",
-          "params": {
-            "threshold": "IGNORE_ALL"
-          }
-        },
-        {
-          "action_id": "health-check",
-          "params": {
-            "app_protocol": "HTTP",
-            "endpoint": "lb-wordpress-http"
-          }
-        },
-        {
-          "action_id": "resilience-tests",
+          "action_id": "cypress",
           "params": {
-            "test_cases": ["testKubernetesPodDelete"]
+            "resources": {
+              "path": "/.vib/wordpress/cypress"
+            },
+            "endpoint": "lb-wordpress-https",
+            "app_protocol": "HTTPS",
+            "env": {
+              "username": "test_user",
+              "password": "ComplicatedPassword123!4"
+            }
```

Comment:
These are set in the `runtime-parameters` file. If it weren't parsed and injected into the pipeline, this action would fail.